pkgs/applications/blockchains/solana/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, rustPlatform
+, IOKit
+, Security
+, AppKit
+, openssl
+, pkg-config
+, protobuf
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "solana-testnet-cli";
+  version = "1.9.2";
+
+  src = fetchFromGitHub {
+    owner = "solana-labs";
+    repo = "solana";
+    rev = "v${version}";
+    sha256 = "sha256-wrv35vBohLztMZPb6gfZdCaXcjj/Y7vnQqINaI6dBM4=";
+  };
+
+  cargoSha256 = "sha256-A5uVa+cRmrkVyw7MFH4QAr0VIFi18wcc2VPFvQyT9EM=";
+
+  buildAndTestSubdir = "cli";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ IOKit Security AppKit ];
+
+  # all the following are only needed for the checkphase
+  # currently checkphase fails on some protobuff compilation
+  # checkInputs = [ pkg-config ];
+  # # Needed to get openssl-sys to use pkg-config.
+  # OPENSSL_NO_VENDOR = 1;
+  # OPENSSL_LIB_DIR = "${openssl.out}/lib";
+  # OPENSSL_DIR="${lib.getDev openssl}";
+  doCheck = false;
```

Comment:
@happysalada I also had some troubles with running protobuf when building lighthouse.
See [here](https://github.com/NixOS/nixpkgs/blob/75a850f80a5d2bf5f9a331f81e089d0483c48daa/pkgs/applications/blockchains/lighthouse/default.nix#L57) for the settings I had to add.

Reply:
Thank you!
I've added the two variables "PROTOC" and "PROTOC_INCLUDE", but it still fails. I'm leaving those commented out so that people can try.
The error is just "No such file or directory". No idea what is making that.

Reply:
Just adding `protobuf` to the `nativeBuildInputs` was enough to make it compile in this case.